src/transitions/transition-ios.ts

```diff
@@ -20,6 +18,8 @@ export class IOSTransition extends PageTransition {
     super.init();
 
     const plt = this.plt;
+    const OFF_RIGHT = plt.isRTL ? '-99.5%' : '99.5%';
+    const OFF_LEFT = plt.isRTL ? '33%' : '-33%';
```

Comment:
Nice! I have someone else testing it today and if all goes well with their tests I will merge it. 🙂